go/couler/conversion/argo_workflow.go

```diff
@@ -0,0 +1,62 @@
+package conversion
+
+import (
+	wfv1 "github.com/argoproj/argo/pkg/apis/workflow/v1alpha1"
+	pb "github.com/couler-proj/couler/go/couler/proto/couler/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const sequentialStepsTemplateSuffix = "main-template"
+
+func convertToArgoWorkflow(workflowPb *pb.Workflow, namePrefix string) (wfv1.Workflow, error) {
```

Comment:
we may also need to verify the input workflow, to make sure it can run in argo

Reply:
Yes that will be verified when I add a submitter, which will be a follow-up work item.